pyanaconda/network.py

```diff
@@ -536,15 +536,6 @@ def get_supported_devices():
     return NetworkDeviceInfo.from_structure_list(network_proxy.GetSupportedDevices())
 
 
-def get_team_devices():
```

Comment:
@rvykydal, are you ok with moving the function `get_team_devices` from  `pyanaconda/network.py`?

Reply:
Yes